### DISCUSSION TITLE
New Window under mouse

### DISCUSSION THREAD

**Discussion by Humorist2601** [anametologin/krohnkite#46] [Ideas]
Hello

Thank you so much for maintaining this extension. I just had one feature request. Hyprland and Polonium both support a feature in which new window appears under the mouse when using Binary Tree layout. Can something similar be implemented in this script?

**Comment by anametologin** [comment]
I have Polonium but do not see this feature. Can you place screenshot where is in Polonium config or video with how this feature works in other ?

**Comment by Humorist2601** [comment]
Sorry I was misremembering the exact setting name. I have attached the screenshot.

Basically what this does is if you have set active window setting to be on mouse hover (kde settings), then you can move your mouse to any window and it will become the active one. Now of you open a new window it will cut the active window in half (in binary tree)

I.e. If you were hovering the right window, the right window will split into two, if you were hovering left window the left window will split into two and so on.

I will try to get a recording once I am home

![Screenshot_20240803_181055](https://github.com/user-attachments/assets/71c875b7-ab8f-4da9-a6b4-ecdb58bff907)

**Comment by anametologin** [comment]
>  I will try to get a recording once I am home

I enabled it in Polonium and it's cool why not. But Krohnkite arranges windows differently, so I need to figure out how to make this work in a way that's appropriate for it.

**Reply by Humorist2601** [reply]
I would love to help you but my job is keeping me very busy nowadays. Thank you so much for maintaining this extension for Plasma 6, it is rock solid stable.

One slight thing, in Polonium, if there are two windows split and you add a new window to one of the sides the new window appears below the active window, whereas in hyprland the window appear underneath the cursor.

I will try somehow to get the screen recordings to differentiate them
